test/index-test.js

```diff
@@ -50,32 +50,49 @@ describe(function() {
       expect(err.exitCode).to.equal(1);
     });
 
-    it('succeeds when one good commit and one bad type-empty commit on branch', async function() {
+    it('fails when one good commit and one bad type-empty commit on branch', async function() {
```

Comment:
shouldn't this pass because we are only looking for one good commit

Reply:
We are only excluding commits that fail both type-empty and subject-empty. If a commit only fails one of those, it is "unexpected" and we let it continue to fail.